packages/react-components/stories/log-table.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { Story, Meta } from '@storybook/react';
```

Comment:
![image](https://user-images.githubusercontent.com/45189696/105649600-cd529900-5eeb-11eb-82b7-d6ef41308883.png)

![image](https://user-images.githubusercontent.com/45189696/105649612-de030f00-5eeb-11eb-809d-90b8b17d8a62.png)

The column size is changing across pages, there is also too much space given for "Level" sometimes. Having a long message in one row also causes all the rows to take more spacing, this causes some side effects like the date being split across multiple lines.

Reply:
Fixed. 
![same width](https://user-images.githubusercontent.com/11761240/105899297-42f85b00-5ff9-11eb-87c1-761b51162d42.gif)